examples/example1xr.html

```diff
@@ -0,0 +1,230 @@
+
+      camera = new THREE.PerspectiveCamera(60, window.innerWidth / window.innerHeight, 0.1, 1000);
+      camera.position.set(0, 1, 2);
+      // camera.lookAt(new THREE.Vector3());
+      scene.add(camera);
+
+      const ambientLight = new THREE.AmbientLight(0x808080);
+      scene.add(ambientLight);
+
+      const directionalLight = new THREE.DirectionalLight(0xFFFFFF, 1);
+      directionalLight.position.set(1, 1, 1);
+      scene.add(directionalLight);
+
+
+      cubeMesh = (() => {
+        const geometry = new THREE.BoxBufferGeometry(.5, .5, .5);
+        const material = new THREE.MeshPhongMaterial({
+          color: 0xff0000,
+        });
+        const mesh = new THREE.Mesh(geometry, material);
+        mesh.rotation.order = 'YXZ';
+        mesh.frustumCulled = false;
+        return mesh;
+      })();
+      scene.add(cubeMesh);
+
+      renderer = new THREE.WebGLRenderer({
+        antialias: true,
+        alpha: true,
+      });
+      // renderer.setPixelRatio(window.devicePixelRatio);
+      renderer.setSize(window.innerWidth, window.innerHeight);
+
+      // window.browser.magicleap.RequestDepthPopulation(true);
+      // renderer.autoClear = false;
+
+      container.appendChild(renderer.domElement);
+    }
+
+
+    function animate(time, frame) {
+      cubeMesh.rotation.x += 0.01;
+      cubeMesh.rotation.y += 0.01;
+
+      renderer.render(scene, renderer.vr.enabled ? renderer.vr.getCamera(camera) : camera);
+    }
+
+    init();
+
+    (async () => {
+      console.log('request device');
+
+      if (navigator.xr) {
+        console.log('request session xr');
+        session = await navigator.xr.requestSession({
+          exclusive: true,
+        }).catch(err => Promise.resolve(null));
+
+        if (session) {
+          const _end = () => {
+            renderer.vr.enabled = false;
+            renderer.vr.setSession(null);
+            renderer.vr.setAnimationLoop(null);
+
+            session.removeEventListener('end', _end);
+          };
+          session.addEventListener('end', _end);
+
+          // console.log('request first frame');
+          session.requestAnimationFrame((timestamp, frame) => {
+            renderer.vr.setSession(session, {
+              frameOfReferenceType: 'stage',
+            });
+
+            const {views} = frame.getViewerPose();
+            const viewport = session.baseLayer.getViewport(views[0]);
+            const width = viewport.width;
+            const height = viewport.height;
+            renderer.setSize(width * 2, height);
+
+            renderer.setAnimationLoop(null);
+
+            renderer.vr.enabled = true;
+            renderer.vr.setSession(session, {
+              frameOfReferenceType: 'stage',
+            });
+            renderer.vr.setAnimationLoop(animate);
+
+            console.log('entered xr');
+          });
+        } else {
+          console.log('no xr displays');
+          renderer.setAnimationLoop(animate);
+        }
+      } else { // WebVR
```

Comment:
I think for these examples we can delete the WebVR entry, since it won't be used by Exokit.

Reply:
👍